host.go

```diff
@@ -695,4 +698,193 @@ func (h *Host) InitCommands(c *chat.Commands) {
 			return nil
 		},
 	})
+
+	c.Add(chat.Command{
+		// TODO: find a better name for reverify
+		// TODO: default for reload
+		// TODO: add keys for a specific duration?
+		// TODO: reverify: what about passphrases?
+		//	- make this a different command (why? a passphrase can't change)
+		//	- who cares, kick them? -- after all, they can just reconnect
+		//	- store a flag in users that authenticated via passphrase and skip here (much more complicated)
+		//  - in which cases does this situation actually happen?
+		// TODO: "panic" (?) command for (import + on + reverify)?
+		// TODO: "print" command with a format for saving to the whitelist file?
+		//   -> hard because the whitelist set only saves fingerprints
```

Comment:
Is there any reason whitelist can't support both? (fingerprints and pubkeys)

Reply:
Not really, but the code using the whitelist set has to be changed (we'd need a new ``Item`` that uses the FP as key and the public  key itself as value). Plus, I just noticed, this would conflict with giving ``add`` fingerprints because we can't generate a key from a fingerprint.